src/commands/init.js

```diff
@@ -30,9 +31,9 @@ const templates = [{
  *
  * @returns {Promise} - Promise for the command.
  */
-export default function init({ parsedArguments, parsedOptions }) {
+export default function init({ parsedArguments, parsedOptions, directory }) {
     const { list, force } = parsedOptions.options;
-    const { name, template, version } = parsedArguments.arguments;
+    const { template, version } = parsedArguments.arguments;
```

Comment:
I believe that removing `name` here will make `roc new` (the alias for `roc init`) to not function correctly.